src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.html

```diff
@@ -49,7 +49,7 @@ <h1 class='slds-text-body_regular'>{CUSTOM_LABELS.commonPaymentServices}</h1>
                                 </p>
                             </template>
                             <template if:false={hasUserDisabledWidget}>
-                                <p class='slds-text-body_regular slds-p-vertical_medium'>{disabledMessage}</p>
+                                <p class='slds-content-message slds-p-vertical_medium'>{disabledMessage}</p>
```

Comment:
I don't see `slds-content-message` applying any styling and couldn't find it in the lightning design system library. Is this custom?